contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -453,31 +462,22 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     /**
      * See {ITeleporterMessenger-getMessageHash}
      */
-    function getMessageHash(
-        bytes32 destinationBlockchainID,
-        uint256 messageID
-    ) external view returns (bytes32) {
-        return sentMessageInfo[destinationBlockchainID][messageID].messageHash;
+    function getMessageHash(bytes32 messageID) external view returns (bytes32) {
+        return sentMessageInfo[messageID].messageHash;
     }
 
     /**
      * @dev See {ITeleporterMessenger-messageReceived}
      */
-    function messageReceived(
-        bytes32 originBlockchainID,
-        uint256 messageID
-    ) external view returns (bool) {
-        return _messageReceived(originBlockchainID, messageID);
+    function messageReceived(bytes32 messageID) external view returns (bool) {
+        return _messageReceived(messageID);
     }
 
     /**
      * @dev See {ITeleporterMessenger-getRelayerRewardAddress}
      */
-    function getRelayerRewardAddress(
-        bytes32 originBlockchainID,
-        uint256 messageID
-    ) external view returns (address) {
-        return _relayerRewardAddresses[originBlockchainID][messageID];
+    function getRelayerRewardAddress(bytes32 messageID) external view returns (address) {
+        return _relayerRewardAddresses[messageID];
```

Comment:
should we require that the message was received before?

Reply:
Yeah I think that makes sense now that we don't require relayer reward addresses to be non-zero